### PR TITLE
fix Unsharp option and add other options such as sharpen and blur

### LINKED ISSUE #202: Unsharp option doesn't work
`unsharp` option doesn't work currently.
the option it doesn't exist in the final command when enabling rf_1.
we need to fix it and add `sharpen` and `blur` as well

### PR DESCRIPTION
**Pull request by sadok-f**
fix Unsharp option and add other options such as sharpen and blur, closes #202

### CODE REVIEW

File: docs/url-options.md [hunk 1]
```diff
@@ -196,10 +196,24 @@ For the hex code, the hash `#` character should be replaced by `%23`
 ### `unsh` : unsharp
 `radiusxsigma{+gain}{+threshold}`
 *Default:* `null`
-*Description:* Sharpens an image with a convolved Gausian operator. A good example `0.25x0.25+8+0.065`.
+*Description:* UnSharpens an image with a convolved Gausian operator. A good example `0.25x0.25+8+0.065`.
 
 **example:`unsh_0.25x0.25+8+0.065`** 
 
+### `sh` : sharp
```
Comment (baamenabar) [inline docs/url-options.md]: it is "sharpen"

File: docs/url-options.md [hunk 2]
```diff
@@ -196,10 +196,24 @@ For the hex code, the hash `#` character should be replaced by `%23`
...
+`radiusxsigma{+gain}{+threshold}`
```
Comment (baamenabar) [inline docs/url-options.md]: sharpen does not have gain and thresold params.

File: docs/url-options.md [hunk 3]
```diff
@@ -196,10 +196,24 @@ For the hex code, the hash `#` character should be replaced by `%23`
...
+*Default:* `null`
+*Description:* Sharpens an image with a convolved Gausian operator. A good example `0.25x0.25+8+0.065`.
```
Comment (baamenabar) [inline docs/url-options.md]: The description would be something  like ... let's merge and I'll update the docs.

File: src/Core/Processor/ImageProcessor.php [hunk 4]
```diff
@@ -285,4 +277,21 @@ protected function updateTargetDimensions(): void
             $this->options->setOption('height', $originalHeight);
         }
     }
+
+    /**
+     * @return string
```
Comment (baamenabar) [inline src/Core/Processor/ImageProcessor.php]: I'm mising a better doc block

Comment (sadok-f) [reply src/Core/Processor/ImageProcessor.php]: yes I'll update that

**Comment by sadok-f** [comment]
Yeah, let's merge it and add proper docs later 👽

**Pull request merged**